.github/workflows/ci.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       env:
         EXPECT_LAVAPIPE: true
       run: |
-          pytest -v examples
+          pytest -vvvs examples
```

Comment:
Are you sure you want to keep these flags as they are before merging?

Reply:
reverting here, might be something for #31